modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -210,6 +210,8 @@ Configuration example:
 
 | `pathRewrite` | Supplies the base path expected by the HTTP listener in your application. This value must begin with `/`. This parameter is used only for applications that are deployed to xref:cloudhub-2::ch2-private-space-about.adoc[private space]. | No
 
+| `instanceType` | Instance type. Only UBP organizations can use this flag. Non-UBP organizations use the `vCores` parameter. See xref:general::pricing.adoc[]. | No
```

Comment:
Will people know what "UBP" means? I don't see it defined anywhere. And do you think we should more about what we mean by instance type in this context?

Reply:
@dmerlob

Reply:
I see UBP in a handful of other docs but have no idea what it means.